### PR TITLE
Added the button on the dashboard to continue the user session

### PR DESCRIPTION
**Pull request by jrosen081**
image

### CODE REVIEW

File: frontend/src/pages/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -1,35 +1,92 @@
...
+const getTitleOfButton = (user: User): string => {
+  switch (user.sessionId) {
+    case -1:
+      return "Begin pre-assessment";
+    case 9:
+      return "Congratulations on finishing the study";
+    default:
+      return (
+        (user.onAssessment ? "Continue session " : "Begin session ") +
+        (user.sessionId + 1)
```
Comment (cindy1u0) [inline frontend/src/pages/Dashboard/Dashboard.tsx:48]: we probably want to stop if sessionID is 8 right or some number

File: frontend/src/firebase/firebaseInteractor.ts [hunk 2]
```diff
@@ -130,9 +130,17 @@ export default class FirebaseInteractor {
...
+    } else {
+      throw new Error("Invalid user");
     }
   }
 
+  async createCurrentUser() {
+    try {
+      this.currentUser = await this.getUser(undefined);
+    } catch (error) {}
+  }
```
Comment (cindy1u0) [inline frontend/src/firebase/firebaseInteractor.ts:145]: maybe u can console out the error or throw an error here? since there's like nothing here for now

**Review by cindy1u0 — APPROVED** [review]

File: frontend/src/firebase/firebaseInteractor.ts (hunk 2 above)
Comment (jrosen081) [reply frontend/src/firebase/firebaseInteractor.ts:145]: Good point. Will do!

File: frontend/src/pages/Dashboard/Dashboard.tsx (hunk 1 above)
Comment (jrosen081) [reply frontend/src/pages/Dashboard/Dashboard.tsx:48]: It does stop (see the `disabled` flag)

**Review by cindy1u0 — APPROVED** [review]

**Pull request merged**